### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by hamoudi-hamoudi**

### CODE REVIEW

File: databases/week2/database_2.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+CREATE INDEX student_name_idx ON student (name);
+
+-- Add a new column to the class table named status 
+ALTER TABLE class ADD status enum('not-started','ongoing','finished');
+
+-- Part 3: More queries
+-- Get all the tasks assigned to users whose email ends in @spotify.com 
+SELECT * FROM user_task INNER JOIN task ON task_id = task.id 
+WHERE user_id = (SELECT id FROM user WHERE email LIKE '%@spotify.com%');
+
```
Comment (Elia-Git) [inline databases/week2/database_2.sql:53]: I am not sure if this query gives you any results. Please test & consider what is really needed to find: Compare your own result with below query:
```
SELECT user.name,user.email, task.title,task.id
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user ON user.id = user_task.user_id
WHERE user.email LIKE '%@spotify.com';
```

File: databases/week2/database_2.sql [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+-- Get all the tasks for 'Donald Duck' with status 'Not started' 
+SELECT * FROM task INNER JOIN user_task ON task_id = task.id 
+WHERE user_id = (SELECT id FROM user WHERE name = 'Donald Duck') 
+AND status_id = (SELECT id from status WHERE name = 'Not started');
+
+-- Get all the tasks for 'Maryrose Meadows' that were created in september 
+SELECT * FROM task INNER JOIN user_task ON task_id = task.id 
+WHERE user_id = (SELECT id FROM user WHERE name = 'Maryrose Meadows') 
+AND MONTH(created)=9 ;
```
Comment (Elia-Git) [inline databases/week2/database_2.sql:62]: Same issue as the above.

File: databases/week2/database_2.sql [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+-- ind how many tasks where created in each month 
+SELECT MONTH(created) AS Month , count(*) AS tasks
+FROM task
+GROUP BY MONTH(created);
```
Comment (Elia-Git) [inline databases/week2/database_2.sql]: Good here: Just a small fix  
```
SELECT MONTHNAME(created) AS Month , count(*) AS tasks
FROM task
GROUP BY MONTH;
```

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Please check out the given feedback! Never commit your code before you make sure you get a result.

File: databases/week2/database_2.sql (hunk 1 above)
Comment (hamoudi-hamoudi) [reply databases/week2/database_2.sql:53]: The task was to find all tasks for this email, and the code works fine its just different way. :grin:
I am not showing the name of the user no one requested it

File: databases/week2/database_2.sql (hunk 2 above)
Comment (hamoudi-hamoudi) [reply databases/week2/database_2.sql:62]: Also the query works :grin:.

**Comment by hamoudi-hamoudi** [comment]
Thank you so much for the review , I will fix the Query

**Pull request merged**